routing/cross_mwm_index_graph.cpp

```diff
@@ -312,7 +335,9 @@ void CrossMwmIndexGraph::GetBorderCross(TRoutingMappingPtr const & mapping, Segm
   // outgoing edge
   vector<Segment> twins;
   GetTwins(s, !isOutgoing, twins);
-  CHECK(!twins.empty(), ("Segment:", s, ", isOutgoing:", isOutgoing));
+  if (twins.empty())
```

Comment:
CrossMwmIndexGraph::GetTwins() was rewritten based on a different principle to work universally and to work with mwm mixture. But sometimes (in osrm case or mixture) it returns an empty result. So the CHECK is exchanged with if.